tests/combined_push/test_combined_push.py

```diff
@@ -445,65 +445,3 @@ def test_do_combined_push_overriden_destination(
     # Ensure the "server" was not called
     mock_starmap_mkt.query_image_by_name.assert_not_called()
     mock_starmap_cmt.query_image_by_name.assert_not_called()
-
-
-@mock.patch("pubtools._marketplacesvm.tasks.community_push.command.Source")
-@mock.patch("pubtools._marketplacesvm.tasks.push.command.Source")
-def test_do_combined_push_overriden_destination_file(
```

Comment:
Had to remove this test as yaml.dump -> yaml.load was having some issues. As far as I could tell it's just how the yaml library is. Still have full coverage so not a huge deal.

Reply:
OK; thanks for info